packages/health/example/lib/main.dart

```diff
@@ -230,7 +231,11 @@ class _HealthAppState extends State<HealthApp> {
   }
 
   Future revokeAccess() async {
-    await health.revokePermissions();
+    try {
```

Comment:
As far as I can tell `revokePermissions` doesn't throw any errors? Or is it because it's not implemented on iOS? In that case shouldn't it be handled by `revokePermissions`?

Reply:
Added an exception to revokePermission that specifies that this is not supported on iOS.